src/dex-ui/pages/dao/NFTDAODashboard/NFTDAODashboardOverview.tsx

```diff
@@ -94,7 +95,7 @@ export function NFTDAODashboardOverview() {
                   labelTextColor={Color.Neutral._500}
                   labelTextStyle="p xsmall medium"
                   labelOpacity="1.0"
-                  value={dao.votingDelay}
+                  value={convertFromBlocksToDays(dao.votingDelay)}
                   valueStyle="p large semibold"
                   valueTextColor={Color.Neutral._900}
                   valueUnitSymbol="days"
```

Comment:
1. When I refresh the Page I see this error
<img width="1247" alt="Screenshot 2023-07-18 at 5 29 25 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/71ccfe52-a7cd-4003-9070-0f79e5689e28">

2. After creating the NFT all NFT fields should be in read mode only same like governance.
<img width="1223" alt="Screenshot 2023-07-18 at 5 31 20 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/34ecdcf6-b139-46be-a714-4b65b8800c52">


3. After Creating the NFT we also show the new nft id created and a link which user can verify in the hashscan it is missing.
<img width="1223" alt="Screenshot 2023-07-18 at 5 31 20 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/a8216edc-9c22-4088-974a-0a24c0fc1d41">

Reply:
Made the requested changes.